starter-code/index.html

```diff
@@ -2,41 +2,94 @@
 <html lang="en">
 <head>
   <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel=stylesheet href=./css/main.css>
   <title>Spotify</title>
 </head>
 <body>
-Premium Discover Help Download
-
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
-
-What’s on Spotify?
-
-Millions of Songs
-There are millions of
-songs on Spotify
-
-HD Music
-Listen to music as if you
-were listening live
-
-Stream Everywhere
-Stream music on your
-smartphone, tablet or
-computer
-
-It’s as yeezy as Kanye West.
-Search
-Know what you want to listen to?
-Just search and hit play.
-Browse
-Check out the latest charts,
-brand new releases and great
-playlists for right now.
-Discover
-Enjoy new music every Monday
-with your own personal playlist.
-Or sit back and enjoy Radio.
+  <header class='site-nav'>
+    <img src=./images/spotify-logo.png class='spo-logo'> 
+    <nav class='navigation'>
+      <ul>
+        <li><a href=#>Premium</a></li>
+        <li><a href=#>Discover</a></li> 
+        <li><a href=#>Help</a></li> 
+        <li><a href=#>Download</a></li>
+      </ul>
+    </nav>
+  </header>
+  <div class='welcome'>
+    <h1 class='everyone'>
+      Music for everyone.
+    </h1>
+    <p class='info'>
+      Spotify is now free on mobile, tablet and computer.
+      <br/>
+      Listen to the right music, wherever you are.
+    </p>
+  </div>
+  <div class='definition'>
+    <div class='what'>
+      <h2>What’s on Spotify?</h2>
+    </div>
+    <div class='million'>
```

Comment:
You can name all three div tags with the same class so that you can reuse css code instead of styling three times the same.